styles.css

```diff
@@ -494,3 +499,403 @@ header nav a {
+  }
+
+  .card-section {
+    grid-column: 1/5;
+    margin-left: 40px;
+  }
+
+  .card-section1 {
+    grid-column: 5/9;
+  }
+
+  .card-section2 {
+    grid-column: 2/8;
+    margin-left: 70px;
+  }
+
+  .grid-ul {
+    display: grid;
+    grid-template-columns: repeat(9, 1fr);
+  }
+
+  .get-intouch h4 {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  .get-intouch form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    margin: 0;
+  }
+
+  .get-intouch input {
+    width: 80%;
+  }
+
+  .get-intouch textarea {
+    width: 80%;
+  }
+}
+
+@media (min-width: 1024px) {
+  main {
+    background-image: url("./assets/images/header_desktop.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    height: 95vh;
+    margin-top: -62px;
+  }
+
+  .flex p {
+    width: 60%;
+  }
+
+  .flex h1 {
+    font-size: 40px;
+    margin: 50px 0 20px 0;
+    padding-top: 0;
+  }
+
+  header {
+    position: relative;
+  }
+
+  header a {
+    display: none;
+  }
+
+  header nav ul {
+    position: absolute;
+    top: 0;
+    left: 30%;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-column-gap: 15px;
+    margin: 0 20px;
+  }
+
+  .about-me {
+    background-image: url('./assets/images/about_me2.png');
+    background-size: 254px 138px;
+    background-repeat: no-repeat;
+    height: 347px;
+    margin-top: 15%;
+    position: relative;
+  }
+
+  .upper-bg {
+    background-image: url('./assets/images/about_me1.png');
+    background-size: 167px 210px;
+    background-repeat: no-repeat;
+    right: -20px;
+    width: 188px;
+    height: 207px;
+    margin-top: -104px;
+    position: absolute;
+  }
+
+  .about-me h4 {
+    font-size: 2.7rem;
+    position: absolute;
+    top: -100px;
+    left: 15%;
+  }
+
+  .about-me button {
+    width: 18%;
+    margin-left: 42%;
+    margin-top: 40px;
+  }
+
+  .about-me p {
+    font-size: 1.3rem;
+    width: 60%;
+    margin: 40px 24% 10px;
+  }
+
+  .grid-ul {
+    margin: 0 20px;
+  }
+
+  .card-section {
+    grid-column: 1/4;
+  }
+
+  .card-section1 {
+    grid-column: 4/7;
+  }
+
+  .card-section2 {
+    grid-column: 7/10;
+    margin-left: 0;
+  }
+
+  .get-intouch {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    margin: 8% 40px 20% 40px;
+  }
+
+  .get-intouch h4 {
+    width: 60%;
+    font-size: 1.7rem;
+  }
+
+  .get-intouch input {
+    width: 80%;
+  }
+
+  .get-intouch textarea {
+    width: 80%;
+  }
+
+  .get-intouch ul {
+    position: absolute;
+    bottom: -300px;
+    left: 35%;
+    padding: 20px;
+  }
+
+  hr {
+    border: 1px solid #dfe1e6;
+  }
+
+  .get-intouch span {
+    display: none;
+  }
+
+  .form-img {
+    background-image: url('./assets/images/contact_form.png');
+    background-repeat: no-repeat;
+    width: 117px;
+    height: 175px;
+    position: absolute;
+    right: 793px;
+    bottom: -188px;
+  }
```

Comment:
- [ ] Kindly the position for this image is not as the Figma design, and you can use the present instead of `px` to make it flexible with you in different designs.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155411823-b0f5c7e9-1a90-40c8-9c13-4c72ce0437e2.png) | ![image](https://user-images.githubusercontent.com/35535130/155411736-18d61f29-2c03-43dc-99c0-94af88898992.png) |